README.md

```diff
@@ -1,65 +1,73 @@
-README
-======
+<p align="center"><a href="https://symfony.com" target="_blank">
+    <img src="http://symfony.com/logos/symfony_black_02.svg">
+</a></p>
 
-What is Symfony?
------------------
-
-Symfony is a PHP 5.3 full-stack web framework. It is written with speed and
-flexibility in mind. It allows developers to build better and easy to maintain
-websites with PHP.
-
-Symfony can be used to develop all kind of websites, from your personal blog
-to high traffic ones like Dailymotion or Yahoo! Answers.
-
-Requirements
-------------
-
-Symfony is only supported on PHP 5.3.9 and up.
-
-Be warned that PHP 5.3.16 has a major bug in the Reflection subsystem and is 
-not suitable to run Symfony (https://bugs.php.net/bug.php?id=62715)
+[Symfony][1] is a **PHP framework** for web applications and a set of reusable
+**PHP components**. Symfony is used by thousands of web applications (including
+BlaBlaCar.com and Spotify.com) and most of the [popular PHP projects][2] (including
+Drupal and Magento).
 
 Installation
 ------------
 
-The best way to install Symfony is to use the [official Symfony Installer][7].
-It allows you to start a new project based on the version you want.
+* [Install Symfony][4] with Composer or with our own installer (see
+  [requirements details][3]).
+* Symfony follows the [semantic versioning][5] strictly, publishes "Long Term
```

Comment:
Personally I don't think it's about installation, rather about way of developing. I think it should be in different section.